perf/benchmark/README.md

```diff
@@ -113,6 +113,7 @@ Required fields to specified via CLI or config file:
 ```bash
 optional arguments:
   -h, --help            show this help message and exit
+  --header HEADER       header:value, can be specified multiple times to add headers \(including Host:\)
```

Comment:
If this is multi valued then it should be 
—headers

Reply:
yeah, you are right.

Reply:
for what its worth the flag is `--header` in curl

Reply:
Either use —header but you can supply the option multiple times, or —headers with all header as a single argument.
This pr was doing a combination